apps/spruce/cypress/integration/waterfall/navigation.ts

```diff
@@ -20,5 +20,8 @@ describe("navigation", () => {
     cy.visit("/commits/evergreen?taskNames=test");
     cy.location("pathname").should("equal", "/project/evergreen/waterfall");
     cy.location("search").should("contain", "tasks=test");
+    cy.visit("/commits/evergreen?buildVariants=Ubuntu");
```

Comment:
@minnakt  I accidentally merged in #791 without adding these changes so I added them to this pr to make sure they aren't missed

Reply:
no problem sounds good